common_types/src/datum.rs

```diff
@@ -552,6 +603,13 @@ impl Datum {
             Datum::Int16(v) => v.to_string(),
             Datum::Int8(v) => v.to_string(),
             Datum::Boolean(v) => v.to_string(),
+            Datum::Date(v) => NaiveDate::from_num_days_from_ce_opt((*v) + EPOCH_DAYS_FROM_CE)
+                .unwrap()
+                .to_string(),
+            Datum::Time(v) => NaiveTime::from_num_seconds_from_midnight_opt(
+                ((*v) / NANOSECONDS) as u32,
+                ((*v) % NANOSECONDS) as u32,
+            ).unwrap().to_string(),
```

Comment:
The format method is complex and weird. Is there any simple method to convert UNIX timestamp to Date?
And does the native date follow the [standard](https://dev.mysql.com/doc/refman/8.0/en/datetime.html) of mysql?

Reply:
Yes, it is weird. there is not a method `from_num_days_from_unix_epoch` in NativeDate; 
Here is some standard or implements:
In MYSQL, the supported date range is '1000-01-01' to '9999-12-31'.
In NativeDate, A 32-bit date representing the elapsed time since CE(0000-01-01) in days (32 bits) , and the supported date range is '262145-01-01' to '262143-12-31'.
In Arrow, A 32-bit date representing the elapsed time since UNIX epoch (1970-01-01) in days (32 bits).

Reply:
I add some comments to explain the logic